code/game/objects/items/nemesis_items.dm

```diff
@@ -0,0 +1,458 @@
+	var/turf/cur_loc = get_turf(src)
+	new /obj/effect/temp_visual/nemesis_circle(cur_loc)
+
+	if(!victim)
+		victim = locate() in cur_loc
+
+	if(victim)
+		victim.apply_damage(75, STAMINA)
+		to_chat(victim, "<span class='userdanger'>You trigger [src] and it explodes in shower of sparks!</span>")
+
+	playsound(cur_loc, 'sound/magic/disable_tech.ogg', 50, TRUE)
+
+	if(glasses && ishuman(glasses.loc))
+		var/mob/living/carbon/human/owner = glasses.loc
+		if(owner.get_item_by_slot(ITEM_SLOT_EYES) != glasses)
+			return
+		glasses.say("Attention: Tactical energy mine in [get_area(cur_loc)] has detonated.")
+
+	qdel(src)
+
+/obj/item/nemesis_mine/proc/trigger_overload()
+	var/turf/cur_loc = get_turf(src)
+	new /obj/effect/temp_visual/nemesis_circle/huge(cur_loc)
+	playsound(cur_loc, 'sound/magic/disable_tech.ogg', 50, TRUE)
+	for(var/mob/living/victim in range(1, cur_loc))
+		to_chat(victim, "<span class='userdanger'>You are hit by an energy blast!</span>")
+		victim.apply_damage(55, STAMINA)
+	qdel(src)
+
+/obj/item/nemesis_mine/throw_at(atom/target, range, speed, mob/thrower, spin=1, diagonals_first = 0, datum/callback/callback, quickstart = TRUE)
+	. = ..()
+	icon_state = "trap_small"
+
+/obj/item/nemesis_mine/throw_impact(atom/hit_atom, datum/thrownthing/throwingdatum)
+	if(!..()) //If not caught, deploy
+		if(overloaded)
+			trigger_overload()
+			return
+		active = TRUE
+		icon_state = "trap_small"
+		update_icon()
+		anchored = TRUE
+		playsound(src, 'sound/magic/blind.ogg', 50, TRUE)
+		if(ishuman(throwingdatum.thrower))
+			var/mob/living/carbon/human/owner = throwingdatum.thrower
+			if(istype(owner.get_item_by_slot(ITEM_SLOT_EYES), /obj/item/clothing/glasses/hud/security/sunglasses/nemesis))
+				glasses = owner.get_item_by_slot(ITEM_SLOT_EYES)
+
+/obj/item/nemesis_mine/emp_act(severity)
+	. = ..()
+	if(. & EMP_PROTECT_SELF)
+		return
+
+	if(prob(35))
+		trigger()
+
+	var/datum/effect_system/spark_spread/sparks = new(get_turf(src))
+	sparks.set_up(4, 0, loc)
+	sparks.start()
+	qdel(src)
+
+/obj/effect/temp_visual/nemesis_circle
+	name = "energy blast"
+	icon = 'icons/obj/nemesis.dmi'
+	icon_state = "trap_explosion"
+	duration = 8
+
+/obj/effect/temp_visual/nemesis_circle/huge
+	name = "big energy blast"
+	icon = 'icons/effects/96x96.dmi'
+	icon_state = "nemesis_circle"
+	duration = 5
+	pixel_x = -32
+	pixel_x = -32
+
+/obj/item/shield/energy/nemesis
+	name = "tactical energy shield"
+	desc = "A tactical wrist-mounted energy shield from Nemesis Solutions."
+	icon = 'icons/obj/nemesis.dmi'
+	icon_state = "nemesis0"
+	base_icon_state = "nemesis"
+	force = 3
+
+	item_flags = SLOWS_WHILE_IN_HAND
+	slowdown = 0
+
+/obj/item/shield/energy/nemesis/Initialize()
+	. = ..()
+	ADD_TRAIT(src, TRAIT_NODROP, CLOTHING_TRAIT)
+
+/obj/item/shield/energy/nemesis/attack_self(mob/living/carbon/human/user)
+	if(clumsy_check && HAS_TRAIT(user, TRAIT_CLUMSY) && prob(50))
+		to_chat(user, "<span class='userdanger'>You beat yourself in the head with [src]!</span>")
```

Comment:
how

why

also, this doesn't return out, so the shield still activates

Reply:
This was taken from the original shield code. Should I remove it?